src/Settings/settings.json

```
@@ -757,5 +640,21 @@
         "default": "default",
         "param": "Core::StyleManager::styleList()",
         "tip": "The style of the whole application."
+    },
+    {
+        "name": "Editor Status",
+        "type": "QMap",
+        "final": "QVariantMap"
+    },
+    {
+        "name": "Language Config",
+        "type": "Object",
+        "sub": [
+            {
+                "name": "Snippets",
```

```suggestion
                "name": "Snippet",
```

The functions should be `querySnippet`, `removeSnippet`, etc. because they only operate on a single snippet.

---

Actually querySnippets return a QStringList, so querySnippets is better. Maybe I can add an entry?

---

> Actually querySnippets return a QStringList, so querySnippets is better. Maybe I can add an entry?

I didn't check that. Then I mean `getSnippets` should be `getSnippet`.

---

Yes. I add a new entry sname for this. The setXXX getXXX removeXXX use the sname and queryXXX use the name.